README.md

```diff
@@ -1,18 +1,10 @@
-## Microsoft Open Source Code of Conduct
-
-This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/).
-For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.
+# ASP.NET Docs
 
-ASP.NET Docs
-============
+This repository contains the conceptual ASP.NET Core and ASP.NET 4.x documentation hosted at [docs.microsoft.com/aspnet](https://docs.microsoft.com/aspnet). Learn more about ASP.NET Core at the [AspNetCore](https://github.com/aspnet/AspNetCore) repository. See the [Contributing Guide](CONTRIBUTING.md) and the [issues list](https://github.com/aspnet/Docs/issues) if you would like to help out.
 
-This project provides the source for [docs.microsoft.com/aspnet](https://docs.microsoft.com/aspnet). You can learn more about ASP.NET Core at the [Home](https://github.com/aspnet/home) repo. See the [Contributing Guide](CONTRIBUTING.md) and the [issues list](\https://github.com/aspnet/Docs/issues) if you would like to help out.
+API PRs (*api/autoapi/...*) shouldn't be made here. Changes to API documentation should be made in the source repository against the triple slash `///` comments.
```

Comment:
This begs the question "which source repository".  If we're going to tell them to go elsewhere for API updates, can we provide more concrete instructions on where to go?  Or perhaps just omit this paragraph altogether?